src/main/java/com/easypost/model/DateDeserializer.java

```diff
@@ -0,0 +1,78 @@
+package com.easypost.model;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class DateDeserializer implements JsonDeserializer<Date> {
+    private static final String[] DATE_FORMATS = new String[]{
```

Comment:
Sigh...

Reply:
Claims API endpoints return a different date format ("yyyy-MM-dd'T'HH:mm:ss") and we can't deserialize it. We need to a custom deserializer